pkg/addon/addon.go

```diff
@@ -844,6 +844,18 @@ func RenderArgsSecret(addon *InstallPackage, args map[string]interface{}) *unstr
 	return u
 }
 
+// deleteArgsSecret delete the addon's args secret file
+func deleteArgsSecret(ctx context.Context, k8sClient client.Client, addonName string) {
+	var sec v1.Secret
+	err := k8sClient.Get(ctx, client.ObjectKey{Namespace: types.DefaultKubeVelaNS, Name: addonutil.Addon2SecName(addonName)}, &sec)
+	if err == nil {
+		err = k8sClient.Delete(ctx, &sec)
+		if err != nil {
+			fmt.Printf("fail to delete %v addon parameter file", addonName)
+		}
+	}
+}
+
```

Comment:
If the error returned from the `get` function is not `errorNotFound`, please return the error.

It's highly recommended to handle this logic using the following pattern:
```
if err := k8sClient.Get(xxx); err == nil {
    // Handle successful get operation
} else {
    // Handle error
    if err != errorNotFound {
        return err
    }
}
```